doc/changelog/10-standard-library/14736-Arith2.rst

```diff
@@ -0,0 +1,7 @@
+- **Deprecated:**
+  Some obsolete files from the ``Arith`` part of the standard library are now explicitely deprecated (``Div2``, ``Even``, ``Gt``, ``Le``, ``Lt``, ``Max``, ``Min``, ``Minus``, ``Mult``, ``NPeano``, ``Plus``).
+  A temporary file ``Arith_prebase`` is introduced to ensure compatibility but should not be imported directly (it will disappear at the end of the deprecation phase).
```

Comment:
```suggestion
  A temporary file ``Arith_prebase`` has been introduced to ensure compatibility but should not be imported directly (it will disappear at the end of the deprecation phase).
```

Reply:
What do you mean by "should not be imported directly"? Perhaps it might be better to say something like:
```suggestion
  A temporary file ``Arith_prebase`` has been introduced to ensure compatibility. Usage is discouraged since it will disappear at the end of the deprecation phase.
```

Reply:
Maybe this?

```suggestion
  ``Arith_prebase`` has been introduced temporarily to ensure compatibility, but it will be removed at the end of the 
  deprecation phase, e.g. in 8.16.
```

Would you add a description of the actions users need to take as a result of this PR?   Seems like it's replace references to the deprecated files with (exactly what?).  And don't use Arith_prebase.  It's a bit odd to draw attention to `Arith_prebase` when we don't want users to import it directly.

Reply:
BTW "explicitely" should be "explicitly"

Reply:
I have tried to give a description of what should be done to avoid deprecation warnings.
Shouldn't the end of deprecation phase be deprecation + 2? thus 8.17 if introduction in 8.15?
Anyway @Alizter and @jfehrle, thanks for your suggestions.

Reply:
I think it's more likely that this PR will get into 8.16 so the EOF should be 8.18.